src/management/RolesManager.js

```diff
@@ -351,8 +351,8 @@ RolesManager.prototype.removePermissions = function(params, data, cb) {
  * @param   {String}    [roleId]          Id of the role
  * @param   {Number}    [params.per_page] Number of results per page.
  * @param   {Number}    [params.page]     Page number, zero indexed.
- * @param   {String}    [params.from]     Optional id from which to start selection.
- * @param   {Number}    [params.take]     The total amount of entries to retrieve when using the from parameter. Defaults to 50.
```

Comment:
This phrasing is not correct; as 'from' is optional (and not used during the initial request w/ checkpoint pagination outside the unique log's endpoint method), `take` can and always will (for the first request) be used without `from`.